bot/university/university.py

```diff
@@ -27,6 +28,119 @@ def __init__(self, bot):
+            except commands.BadArgument:
+                modules_error.append(module_upper)
+
+        embed = _create_embed_module_roles(modules_added, modules_removed, modules_error)
+        await ctx.author.send(embed=embed)
+
+    @toggle_module.command(name="add", hidden=True)
+    @commands.is_owner()
+    @command_log
+    async def add_module_role(self, ctx: commands.Context, module_name: str):
+        """Command Handler for the `module` subcommand `add`.
+
+        Allows the bot owner to add a specific role to the module roles.
+
+        Args:
+            ctx (discord.ext.commands.Context): The context in which the command was called.
+            module_name (str): The name of the role which should be added.
+        """
+        module_role = await commands.RoleConverter().convert(ctx, module_name.upper())
+
+        try:
+            self._db_connector.add_module_role(module_role.id)
+            await ctx.send(f"Die Rolle \"**__{module_role}__**\" wurde erfolgreich zu den verfügbaren Modul-Rollen "
+                           f"hinzugefügt.",
+                           delete_after=8)
+        except IntegrityError:
+            await ctx.send(f"Die Rolle \"**__{module_role}__**\" gehört bereits zu den verfügbaren Modul-Rollen.",
+                           delete_after=8)
+
+    @add_module_role.error
+    async def add_module_role_error(self, ctx: commands.Context, error: commands.CommandError):
+        """Error Handler for the `module` subcommand `add`.
+
+        Handles specific exceptions which occur during the execution of this command. The global error handler will
+        still be called for every error thrown.
+
+        Args:
+            ctx (discord.ext.commands.Context): The context in which the command was called.
+            error (commands.CommandError): The error raised during the execution of the command.
+        """
+        if isinstance(error, commands.BadArgument):
+            await _module_role_error(ctx, error)
+
+    @toggle_module.command(name="remove", hidden=True)
+    @commands.is_owner()
+    @command_log
+    async def remove_module_role(self, ctx: commands.Context, module_name: str):
+        """Command Handler for the `module` subcommand `remove`.
+
+        Allows the bot owner to remove a specific role from the module roles.
+
+        Args:
+            ctx (discord.ext.commands.Context): The context in which the command was called.
+            module_name (str): The name of the role which should be removed.
+        """
+        module_role = await commands.RoleConverter().convert(ctx, module_name.upper())
+
+        self._db_connector.remove_module_role(module_role.id)
+        await ctx.send(f"Die Rolle \"**__{module_role}__**\" wurde aus den verfügbaren Modul-Rollen entfernt.",
+                       delete_after=8)
+
+    @remove_module_role.error
+    async def remove_module_role_error(self, ctx: commands.Context, error: commands.CommandError):
```

Comment:
Why 2 functions if they do the same? Just put the annotation
    @remove_module_role.error
and 
    @add_module_role.error
 
onto the same method

Reply:
I literally tried that before and I swear IntelliJ showed me an error. Now it works. 😫

Reply:
Oh, I think the annotation didn't work because the error handler was to high up in the file. Oops.